h/models/user.py

```diff
@@ -194,6 +194,8 @@ def __table_args__(cls):  # noqa: N805
     #: A NULL value in this column indicates the user has never accepted a privacy policy.
     privacy_accepted = sa.Column(sa.DateTime, nullable=True)
 
+    identities = sa.orm.relationship("UserIdentity", backref="user", cascade="all, delete-orphan")
```

Comment:
The `cascade="all, delete-orphan"` here is what tells sqlalchemy that it needs to delete the corresponding `UserIdentity`'s when a `User` gets deleted or when `UserIdentity`'s get removed from a `User`'s `.identities`.

Reply:
This and another comment on this thread are both really useful information. Given that it's really documenting SQLAlchemy and not our code per se, they probably don't make sense as comments, but I wonder if a SO post would be useful on some of this stuff?